db/migrate/20230307161705_add_address_to_user.rb

```diff
@@ -0,0 +1,8 @@
+class AddAddressToUser < ActiveRecord::Migration[7.0]
+  def change
+    add_column :users, :address, :string
+    add_column :users, :country, :string
+    add_column :users, :city, :string
+    add_column :users, :post_code, :string
```

Comment:
is post_code not usually an integer?

Reply:
(but maybe this is a german thing)

Reply:
england has letters

Reply:
i guess it is a GRAY area (see what i did there )

fine by me then